napari/components/layerlist.py

```diff
@@ -123,6 +124,36 @@ def _extent_world(self) -> np.ndarray:
         """
         return self._get_extent_world([layer.extent for layer in self])
 
+    def _get_min_and_max(self, mins_list, maxes_list):
+
+        # Reverse dimensions since it is the last dimensions that are
+        # displayed.
+        mins_list = [mins[::-1] for mins in mins_list]
+        maxes_list = [maxes[::-1] for maxes in maxes_list]
+
+        with warnings.catch_warnings():
+            # Taking the nanmin and nanmax of an axis of all nan
+            # raises a warning and returns nan for that axis
+            # as we have do an explict nan_to_num below this
```

Comment:
@grlee77 do you want to move the nan_to_num call to this function rather than leaving it outside of the function? Otherwise the comment feels obscure and incorrect.

Reply:
Yeah, I think the other half of the if/else statement (where the layer list is empty) was also using the `nan_to_num` to fill in the values, but if we move `nan_to_num` into this function we can just handle that case by setting the values directly.